bot/resources/tags/empty-json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
+A JSON may never be completely empty and must always at least have one of the following items.
+
+```
+object
```

Comment:
An object is the dict-like structure that you can have in a JSON, yes? I think it would be easier for most of the intended readers to describe the valid structures in terms of their Python analogues, since the goal isn't to teach anyone anything about JSONs beyond how to debug this error.

Reply:
In the current text below it, says it's common to use {} to make an object, which is similar to a dictionary in python. Should I move this line up?

Reply:
Even if it is the correct term, I am also of the mindset that we use python keywords. The tags in my opinion are by no means an extensive resource, but more of a quick overview to explain a concept, and set someone on the right path of looking into it.

The source you included uses `object`. If someone is interested in learning more about JSON, they can follow that and learn more. It does not make sense to have it that far removed from python though when the original question is probably in python.

Reply:
> Even if it is the correct term, I am also of the mindset that we use python keywords. The tags in my opinion are by no means an extensive resource, but more of a quick overview to explain a concept, and set someone on the right path of looking into it.
> 
> The source you included uses `object`. If someone is interested in learning more about JSON, they can follow that and learn more. It does not make sense to have it that far removed from python though when the original question is probably in python.

Ah right now I understand what you are saying. In the discord chat I didn't completely understand what you were saying. Now with context I understand more. I will fix this.